codec-redis/src/main/java/io/netty/handler/codec/redis/FixedRedisMessagePool.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The Netty Project
+ * Copyright 2021 The Netty Project
```

Comment:
```suggestion
 * Copyright 2016 The Netty Project
```

We don't change copyright years